### PR TITLE
MOYO header

### PR DESCRIPTION
**Pull request by abcdesign1**
- [DEMO LINK](https://vadym4che.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://vadym4che.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,29 @@
...
+          <li class="nav__item"><a href="#samsung">SAMSUNG</a></li>
+          <li class="nav__item"><a href="#smartPhones">SMARTPHONES</a></li>
+          <li class="nav__item">
+            <a
+              href="#laptopDesktop"
+              data-qa="hover"
+            >
+            LAPTOPS & COMPUTERS
+            </a>
+          </li>
```
Comment (Andr1yK) [inline src/index.html:28]:
```suggestion
              data-qa="hover"
            >
            	LAPTOPS & COMPUTERS
            </a>
          </li>
```

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,29 @@
...
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="is-active">APPLE</a></li>
+          <li class="nav__item"><a href="#samsung">SAMSUNG</a></li>
+          <li class="nav__item"><a href="#smartPhones">SMARTPHONES</a></li>
+          <li class="nav__item">
+            <a
+              href="#laptopDesktop"
+              data-qa="hover"
+            >
+            LAPTOPS & COMPUTERS
```
Comment (Andr1yK) [inline src/index.html:26]: let's use `text-transform` to make this text uppercase

https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

File: src/style.css [hunk 3]
```diff
@@ -1 +1,64 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
```
Comment (Andr1yK) [inline src/style.css:1]: better to use <link> tag

https://stackoverflow.com/questions/12316501/including-google-fonts-link-or-import

File: src/style.css [hunk 4]
```diff
@@ -1 +1,64 @@
...
+
+body,
+ul {
+  margin: 0;
+}
+
+ul {
+  padding: 0;
+}
```
Comment (Andr1yK) [inline src/style.css:10]: 🔥

File: src/style.css [hunk 5]
```diff
@@ -1 +1,64 @@
...
+.header {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+#logo {
```
Comment (Andr1yK) [inline src/style.css:21]: wow wow wow... what happened if I'll want to add one more logo?)

File: src/style.css [hunk 6]
```diff
@@ -1 +1,64 @@
...
+
+.nav__item a:hover {
+  color: #00acdc;
+}
+
+.nav__item a.is-active {
+  color: #00acdc;
+  border-bottom: 4px solid #00acdc;
+  border-top: 4px solid transparent;
+}
```
Comment (Andr1yK) [inline src/style.css:64]: let's create class for link

File: src/style.css (hunk 4 above)
Comment (vadym4che) [reply src/style.css:10]: this approach was invented at one Q&A in order to force successful test passing and deployment of this task)

**Pull request closed without merging**